exer_5.py

```diff
@@ -0,0 +1,14 @@
+Numbers = [9, 4, 8, 10, 2, 4, 8, 3, 14, 4, 8]
```

Comment:
Convert variable name to lowercase.

Check python standards for the naming convention of function, variables, classes, and const, etc.